### PR TITLE
Synced qa-1201-b2 branch with latest state from qa-1201-b2 env

### PR DESCRIPTION
**Pull request by akilalakshmanan**

### CODE REVIEW

File: deployment/v3/mosip/pms/install.sh [hunk 1]
```diff
@@ -34,7 +34,7 @@ function installing_pms() {
   helm -n $NS install pms-policy mosip/pms-policy --set istio.corsPolicy.allowOrigins\[0\].prefix=https://$PMP_HOST --version $CHART_VERSION
 
   echo Installing pmp-ui
-  helm -n pms install pmp-ui mosip/pmp-ui  --set pmp.apiUrl=https://$INTERNAL_API_HOST/ --set istio.hosts=["$PMP_HOST"] --set imge.repository=mosipdev/pmp-ui --version 12.0.2
+  helm -n pms install pmp-ui mosip/pmp-ui  --set pmp.apiUrl=https://$INTERNAL_API_HOST/ --set istio.hosts=["$PMP_HOST"]
```
Comment (ckm007) [inline deployment/v3/mosip/pms/install.sh:37]: Chart version not mentione?

File: deployment/v3/mosip/regproc/install.sh [hunk 2]
```diff
@@ -43,7 +43,7 @@ function installing_regproc() {
   helm -n $NS install regproc-group2 mosip/regproc-group2  --version $CHART_VERSION
 
   echo Installing group3
-  helm -n $NS install regproc-group3 mosip/regproc-group3  --version $CHART_VERSION
+  helm -n $NS install regproc-group3 mosip/regproc-group3 --set image.tag=1.2.0.1-B2 --version $CHART_VERSION
```
Comment (ckm007) [inline deployment/v3/mosip/regproc/install.sh:46]: This is not required as image tag is 1.2.0.1-B2 for 1.2.0.1-beta release.

File: deployment/v3/mosip/pms/install.sh (hunk 1 above)
Comment (akilalakshmanan) [reply deployment/v3/mosip/pms/install.sh:37]: chart version is already 12.0.1-B2 which is deployed in qa-1201-b2 env sir

Comment (ckm007) [reply deployment/v3/mosip/pms/install.sh:37]: I dont see the version in this command:

helm -n pms install pmp-ui mosip/pmp-ui  --set pmp.apiUrl=https://$INTERNAL_API_HOST/ --set istio.hosts=["$PMP_HOST"]

**Review by ckm007 — APPROVED** [review]

**Pull request merged**